src/server.prod.js

```diff
@@ -45,21 +48,37 @@ class ProdServer {
     const nockPath = this.config.nockPath;
     const accessLogs = this.config.accessLogs || 'clf';
     const renderFn = this.getRenderFunction();
+    const compressionMiddleware = this.config.compressHTML ? compression : () => (...[,,next]) => next();
     if (!this.config.isCDN) {
       this.setupStaticFiles(app);
     };
     app.use('/healthcheck', healthcheck());
-    app.use(...getMiddleWares({ renderFn, before, after, nock, nockPath, accessLogs }));
+    app.use(...getMiddleWares({
+      renderFn,
+      before,
+      after: [
+        ...after,
+        compressionMiddleware({
```

Comment:
It would be great if we could add the compression middleware to the array conditionally, then we would not need to define and add this dummy middleware.

Reply:
Changed.